src/extension/features/general/import-notification/index.js

```diff
@@ -73,8 +73,10 @@ export class ImportNotification extends Feature {
         if (
           (typeof account.getDirectConnectEnabled === 'function' &&
             account.getDirectConnectEnabled()) ||
+          account.directConnectEnabled ||
```

Comment:
This specific getter doesn't exist in my version of YNAB, but I don't know if it's safe to delete. If it is the whole if can be simplified to `if (account.isDirectImportActive)`.